src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
```

Comment:
It isn't a good approach to add styling based purely on the tag type, except for html and body tags.
Try removing this entry and dealing with "p" when needed in each block